jena-fuseki2/jena-fuseki-main/src/main/java/org/apache/jena/fuseki/main/FusekiServer.java

```diff
@@ -873,6 +875,16 @@ public Builder realm(String realm) {
             return this;
         }
 
+//        /**
```

Comment:
Couldn't this be uncommented?

Also type `Berarer` in the javadoc here

Reply:
Bear authentication isn't support in the server builder. The key issue is how custom code gets added to a server installation.  It is not clear yet whether there can be server builder support or whether a Fuseki module is better.

So not in the public API at this time.

In the PR, the server builder does not do anything with `bearerVerifiedUser`. It is paired with `auth(AuthScheme.BEARER)` causing an exception.

javadoc fixed.